accessibility/index.html

```diff
@@ -0,0 +1,758 @@
+                    <pre><code data-trim contenteditable> 
+                    
+<!-- Link Example -->
+ <a href="http://www.someurl.com">Go to some Web page</a><br/><br/>
+
+ <!-- Button Example -->
+ <button>Click me</button>
+ 
+                    </code></pre>
+                    <a href="#" style="color:blue;font-size:14pt">Go to some Web page</a><br/><button style="width:150px;height:40px;font-size:14pt">Click me</button> 
+				</section>
+                
+                <section>
+                    <h2><span class="gdi">Non-Semantic</span> Elements</h2>
+                    <small>Divs styled with CSS to look semantic</small>
+                    <pre><code data-trim contenteditable>    
+                    
+ <!-- Link Example -->
+ <div class="link-class">Go to some Web page</div>
+
+ <!-- Button Example -->
+ <div class="button-class">Click Me</div> 
+ 
+                    </code></pre>
+                    
+                    <div class="link-class" style="font-size:14pt">Go to some Web page</div>  
+                    <br/>
+                    <a href="https://inclusiveThinking.jsbin.com/dibevo/edit?html,css,output" target="_blank">Non-Semantic Button</a>
+                    
+          
+                    
+                </section>
+  
+                <!-- EXAMPLES OF SEMATIC VS NON-SEMANTIC HTML END -->  
+            
+                <section>
+                    <p>The following exercises illustrate how semantic HTML elements and non-semantic elements can look the same but are interpreted differently by screen readers</p>
+                </section
+				
+				<!-- Exercise 01 START -->
+
+				<section>
+					<h3>Exercise One</h3>
+          			<p>Use VoiceOver to read content that uses <br/><strong>Non-Semantic</strong> HTML elements</p>
+		  			<small><a href="https://inclusiveThinking.jsbin.com/mudiwu/1/edit?html,css,output" target="_blank">Open Exercise</a></small>
+                    <aside class="notes">
+						Open Code in JS Bin's Live Preview before using screen reader
+					</aside>
+				</section>
+
+				<!-- Exercise 01 END -->
+
+				<!-- Exercise 02 START -->
+
+				<section>
+					<h3>Exercise 2</h3>
+          			<p>Modify the HTML using <strong>Semantic</strong> HTML elements <br/>and read the page again</p>
+		  			<small><a href="https://inclusiveThinking.jsbin.com/dihebu/3/edit?html,css,output" target="_blank">Modify HTML from Exercise One</a></small>
+                     <aside class="notes">
+						Have students modify the HTML using semantic elements then open Code in JS Bin's Live Preview before using screen reader. Students will now be hear a difference in the way the HTML is identified by a screen reader
+					</aside>
+				</section>
+				</section>
+
+				<!-- Exercise 02 END -->
+                    
+                <section>
+                    <h2>Using <span class="gdi">Browser Plugins</span> for <span class="gdi">Accessibility Testing</span></h2>
```

Comment:
This is an awesome slide! Might be worthwhile to break it up into several sections